src/azure-cli/azure/cli/command_modules/netappfiles/custom.py

```diff
@@ -349,12 +360,12 @@ def create_backup_policy(client, resource_group_name, account_name, backup_polic
         yearly_backups_to_keep=yearly_backups,
         enabled=enabled,
         tags=tags)
-    return client.create(resource_group_name, account_name, backup_policy_name, body)
+    return client.begin_create(resource_group_name, account_name, backup_policy_name, body)
 
 
 def patch_backup_policy(client, resource_group_name, account_name, backup_policy_name, location=None,
-                        daily_backups=None, weekly_backups=None, monthly_backups=None,
-                        yearly_backups=None, enabled=False, tags=None):
+                        daily_backups=0, weekly_backups=0, monthly_backups=0,
+                        yearly_backups=0, enabled=False, tags=None):
```

Comment:
Why not use `None` as default value?

Reply:
Hmmm this must have been a merge conflict gone wrong. You are right, this should be None. Will fix this now.